src/main/scala/uk/gov/hmrc/hmrcfrontend/views/Utils.scala

```diff
@@ -24,6 +25,12 @@ trait Utils extends UtilsSupport {
     path
       .map(p => s"$p/$file")
       .getOrElse(uk.gov.hmrc.hmrcfrontend.controllers.routes.Assets.at(file).url)
+
+  private[views] def maybeBackLinkDataModule(href: String, backLinkConfig: BackLinkConfig): Option[String] =
+    (href match {
+      case unset if unset == "" || unset == "#" => Some(""" data-module="hmrc-back-link"""")
+      case _                                    => None
+    }).filter(_ => backLinkConfig.enableBrowserBack)
```

Comment:
basically, if the `href` is `#` or blank, and the feature is enabled, add the `data-module` attribute to trigger the JS in hmrc-frontend

Reply:
removed this now